pinc/page_table.inc

```diff
@@ -548,6 +531,39 @@ function changeSelection(sel) {
     }
 }
 
+/**
+ * Can the currently logged in user see the usernames of users who have
+ * worked on a particular page in a project?
+ *
+ *
+ * @param Project $project
+ * @param array $page_res
+ *   The page data returned by `fetch_page_table_data`
+ * @param Round[] $rounds_to_display
+ *   Which rounds to check for user work.
+ *
+ * @return bool
+ */
+function can_user_see_usernames_for_page(Project $project, array $page_res, array $rounds_to_display): bool
+{
+    global $pguser;
```

Comment:
`User::current_username()` is preferable. Death to (most) globals.

Reply:
Huzzah! :)

Done.